src/pages/404/index.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react'
 
-const NotFoundPage = () => (
-  <div>
-    <h1>NOT FOUND</h1>
-    <p>You just hit a route that does not exist... the sadness.</p>
-  </div>
-)
+import Layout from 'shared/components/layout'
+
+const NotFoundPage = ({ pageContext }) => {
+  return (
+    <Layout intlInfo={ pageContext }>
+      <h1>NOT FOUND</h1>
+      <p>You just hit a route that does not exist... the sadness.</p>
```

Comment:
This is not related to this PR, but these messages should be translated.

Reply:
Will do it soon.